.codespellignore

```diff
@@ -1,2 +1,3 @@
 ede
 EDE
+prefered
```

Comment:
I know you used it as a variable name, but I think the correct spell is `preferred` with `rr`.

Reply:
This is a typo that exists in the Linux networking stack - we have not invented the spelling issue ourselves, see, e.g. [torvalds/linux:include/uapi/linux/if_addr.h#L61](https://github.com/torvalds/linux/blob/684a64bf32b6e488004e0ad7f0d7e922798f65b6/include/uapi/linux/if_addr.h#L61)

Reply:
Oh... I didn't know that.